services/src/main/java/org/fao/geonet/api/records/backup/DownloadBackup.java

```diff
@@ -0,0 +1,124 @@
+package org.fao.geonet.api.records.backup;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import jeeves.server.context.ServiceContext;
+import jeeves.server.dispatchers.ServiceManager;
+import org.fao.geonet.ApplicationContextHolder;
+import org.fao.geonet.api.API;
+import org.fao.geonet.api.ApiParams;
+import org.fao.geonet.api.ApiUtils;
+import org.fao.geonet.exceptions.JeevesException;
+import org.fao.geonet.kernel.GeonetworkDataDirectory;
+import org.fao.geonet.utils.Log;
+import org.quartz.Scheduler;
+import org.quartz.Trigger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.ApplicationContext;
+import org.springframework.core.io.FileSystemResource;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.stereotype.Controller;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.bind.annotation.*;
+
+import javax.servlet.http.HttpServletRequest;
+import java.io.File;
+import java.io.FileFilter;
+import java.nio.file.Path;
+
+import static org.fao.geonet.api.ApiParams.API_CLASS_RECORD_OPS;
+import static org.fao.geonet.api.ApiParams.API_CLASS_RECORD_TAG;
+import static org.quartz.TriggerBuilder.newTrigger;
+
+@RequestMapping(value = {
+        "/api/records",
+        "/api/" + API.VERSION_0_1 +
+                "/records"
+})
+@Api(value = API_CLASS_RECORD_TAG,
+        tags = API_CLASS_RECORD_TAG,
+        description = API_CLASS_RECORD_OPS)
+
+@Controller
+public class DownloadBackup {
+
+    @Autowired
+    ServiceManager serviceManager;
```

Comment:
Have you check this `Autowired` works fine in multi-node? As far as I understood, due to the multi-node stuff most of the beans require to be retrieved explicitly in the code with `context.getBean()`